app/app.go

```diff
@@ -26,6 +26,7 @@ type Config struct {
 	HTTPSListenPort int
 	K8sMode         string
 	Debug           bool
+	NoCACert        bool
```

Comment:
can you change everything to add the `s` at the end.  So `NoCACerts`, `--no-cacerts`